app/models/artist.rb

```diff
@@ -1,2 +1,5 @@
 class Artist < ApplicationRecord
+  has_many :concerts
+  has_many :followed_artists
+  has_many :artist_genres
```

Comment:
I think it's `has_many :genres through :artist_genres`, because the artist_genres table is a JOIN table @isaquito1993